tezos/context/src/working_tree/storage.rs

```diff
@@ -204,6 +205,12 @@ pub enum StorageError {
     RootOfInodeNotAPointer,
 }
 
+impl std::fmt::Display for StorageError {
```

Comment:
Please add error annotations with text to each enum entry instead

Reply:
@tizoc Done